guidelines/index.html

```diff
@@ -354,22 +354,28 @@ <h2 id="w3c-recommendation-05-june-2018"><abbr title="World Wide Web Consortium"
-          
-          
-        </p><p>この文書は、<a id="w3c_process_revision" href="https://www.w3.org/2018/Process-20180201/">1 February 2018 <abbr title="World Wide Web Consortium">W3C</abbr> Process Document</a> に従っている。
-        </p></section>
+        <em>この節では、この文書の発行された時点でのステータスを説明する。現行の <abbr title="World Wide Web Consortium">W3C</abbr> の発行文書、及び、この技術レポートの最新版は、https://www.w3.org/TR/ にある <a href="https://www.w3.org/TR/"><abbr title="World Wide Web Consortium">W3C</abbr> technical reports index</a> で参照可能である。</em>
+      </p>
+
+        <p>これは、<a href="https://www.w3.org/WAI/GL/">Accessibility Guidelines Working Group</a> が作成した WCAG 2.1 の <a href="https://www.w3.org/2018/Process-20180201/#RecsW3C">Recommendation</a> である。これは <a href="https://www.w3.org/WAI/WCAG21/errata/">errata</a> を取り入れており、<a href="#changelog">変更点</a>で説明されている。ある時点で、追加の変更が <a href="https://www.w3.org/2018/Process-20180201/#revised-rec">Edited or Amended Recommendation</a> へ取り入れられる可能性がある。</p>
+
+        <p>コメントするためには、<a href="https://github.com/w3c/wcag/issues/new"><abbr title="World Wide Web Consortium">W3C</abbr> WCAG GitHub リポジトリにイシューを提出していただきたい</a>。この文書で提案されている達成基準は、議論の追跡に関するイシューを参照しているが、ワーキンググループは、パブリックコメントを新しいイシューとして提出するよう要請する。GitHub アカウントを作成してイシューを提出することは自由である。GitHub でイシューの提出が実現できない場合、<a href="mailto:public-agwg-comments@w3.org?subject=WCAG%202.1%20public%20comment">public-agwg-comments@w3.org</a> (<a href="https://lists.w3.org/Archives/Public/public-agwg-comments/">comment archive</a>) へ電子メールを送付していただきたい。</p>
+
+        <p>この文書は <a href="https://www.w3.org/WAI/GL/">Accessibility Guidelines  Working Group</a> によって、<a href="https://www.w3.org/2023/Process-20230612/#recs-and-notes">Recommendation track</a> を使用した勧告として公開された。
+        </p>
+
+        <p><abbr title="World Wide Web Consortium">W3C</abbr> は、この仕様を Web 標準として広く展開することを推奨する。</p>
+
+        <p><abbr title="World Wide Web Consortium">W3C</abbr> 勧告は、広範な合意形成を経て <abbr title="World Wide Web Consortium">W3C</abbr> とその会員によって承認された仕様であり、ワーキンググループのメンバーから実装に対する <a href="https://www.w3.org/Consortium/Patent-Policy/#sec-Requirements">royalty-free licensing</a> が約束されている。
+        </p>
+
+        <p>この文書は <a href="https://www.w3.org/Consortium/Patent-Policy-20170801/">1 August 2017 <abbr title="World Wide Web Consortium">W3C</abbr> Patent Policy</a> の下で活動するグループによって作成された。<abbr title="World Wide Web Consortium">W3C</abbr> では、ワーキンググループの成果物に関係する <a rel="disclosure" href="https://www.w3.org/2004/01/pp-impl/35422/status">public list of any patent disclosures</a> を管理しており、そのページには特許開示にあたっての指示も含まれている。<a href="https://www.w3.org/Consortium/Patent-Policy/#def-essential">Essential Claim(s)</a> を含む特許について実際に知識のある人は、<a href="https://www.w3.org/Consortium/Patent-Policy/#sec-Disclosure">section 6 of the <abbr title="World Wide Web Consortium">W3C</abbr> Patent Policy</a> に従って、その情報を開示することが求められる。
```

Comment:
このへん、actual knowledge などは特許関連の専門用語ぽいのでちゃんと確認した方がいいかもしれません (漠然としたコメント)。

Reply:
ここの話は、ざっくりいうと「このW3Cの勧告と既存の特許がバッティングするおそれがある場合に、特許権者 (もしくはその特許について知っているる者) は申し出をしてほしい」ということ。

- individualを「人」としているのは柔らかすぎるかも。通常「個人」と訳されるが、ここでは「者」くらいでいいような気がする
- "section 6 of the W3C Patent Policy" は1つの固有名詞ではなく、「W3C Patent Policy」という文書の「6. Disclosure」という見出しを指しているので少し訳した方がいいかも

こうかしら……?

```suggestion
        <p>この文書は <a href="https://www.w3.org/Consortium/Patent-Policy-20170801/">1 August 2017 <abbr title="World Wide Web Consortium">W3C</abbr> Patent Policy</a> の下で活動するグループによって作成された。<abbr title="World Wide Web Consortium">W3C</abbr> では、ワーキンググループの成果物に関係する <a rel="disclosure" href="https://www.w3.org/2004/01/pp-impl/35422/status">public list of any patent disclosures</a> を管理しており、そのページには特許開示にあたっての指示も含まれている。ある特許について実際の知識を持っている者が、その特許に <a href="https://www.w3.org/Consortium/Patent-Policy/#def-essential">Essential Claim(s)</a> が含まれていると判断した場合、<a href="https://www.w3.org/Consortium/Patent-Policy/#sec-Disclosure"><abbr title="World Wide Web Consortium">W3C</abbr> Patent Policy のsection 6</a> に従って、その情報を開示することが求められる。
```

Reply:
すこし真面目に調べると、
現実の認識（actual knowledge）
https://www.mext.go.jp/b_menu/shingi/bunka/gijiroku/013/06073103/003/008.htm
とされてますね…

「Patent Policy の section 6」は苦しいところですが、「の」を除くとそれはそれで微妙かと思いますので、いただいた案を反映するとこうですか。

```suggestion
        <p>この文書は <a href="https://www.w3.org/Consortium/Patent-Policy-20170801/">1 August 2017 <abbr title="World Wide Web Consortium">W3C</abbr> Patent Policy</a> の下で活動するグループによって作成された。<abbr title="World Wide Web Consortium">W3C</abbr> では、ワーキンググループの成果物に関係する <a rel="disclosure" href="https://www.w3.org/2004/01/pp-impl/35422/status">public list of any patent disclosures</a> を管理しており、そのページには特許開示にあたっての指示も含まれている。ある特許について現実の認識を持っている者が、その特許に <a href="https://www.w3.org/Consortium/Patent-Policy/#def-essential">Essential Claim(s)</a> が含まれていると判断した場合、<a href="https://www.w3.org/Consortium/Patent-Policy/#sec-Disclosure"><abbr title="World Wide Web Consortium">W3C</abbr> Patent Policy の section 6</a> に従って、その情報を開示することが求められる。
```

Reply:
actual knowledgeは私も調べたのですが、どうも固定の定訳があるという感じでもないようで、「現実の認識を持っている」としてしまうと「現実を認識していること」と読めてしまうのでよくないように感じています。
現実の認識を持っている→現実に認識を持っている、とするのはありかもしれません。

Reply:
固定訳はなさそうですが、「現実に認識」ですとそれはニュアンスが変わりそうな…ということで動詞を少し固くして「有する」とするのはどうでしょうか

```suggestion
        <p>この文書は <a href="https://www.w3.org/Consortium/Patent-Policy-20170801/">1 August 2017 <abbr title="World Wide Web Consortium">W3C</abbr> Patent Policy</a> の下で活動するグループによって作成された。<abbr title="World Wide Web Consortium">W3C</abbr> では、ワーキンググループの成果物に関係する <a rel="disclosure" href="https://www.w3.org/2004/01/pp-impl/35422/status">public list of any patent disclosures</a> を管理しており、そのページには特許開示にあたっての指示も含まれている。ある特許について現実の認識を有する者が、その特許に <a href="https://www.w3.org/Consortium/Patent-Policy/#def-essential">Essential Claim(s)</a> が含まれていると判断した場合、<a href="https://www.w3.org/Consortium/Patent-Policy/#sec-Disclosure"><abbr title="World Wide Web Consortium">W3C</abbr> Patent Policy の section 6</a> に従って、その情報を開示することが求められる。
```

Reply:
actual knowledge は constructive knowledgeと対比される言葉で、 constructive knowledge の方は「知っていて当然だ」「立場上、知っているはずだ」「知っておく義務があるはずだ」というような状況において「知っているとみなされる」というニュアンスのようです。
それに対して actual knowledge は、「本当に知っている」「実際に知っている」というニュアンスになります。

ですので、「現実に認識を有する」という言い方でニュアンスとして問題ないと思います。「実際に知識を有する」というような訳でも良いかと思います。

Reply:
@bakera 

了解です。「現実に認識を有する」としてpushしました。